src/lib/crudella.ts

```diff
@@ -0,0 +1,24 @@
+import { bootstrapConfiguration } from './service/bootstrap';
+import { createHandlers } from './service/handlers';
+import { Definitions } from './settings/definitions';
+
+export const createService = <T extends { id: any }, C extends object>(defs: Definitions<T>) => {
```

Comment:
I generally disagree with this requirement for the object with must-have id attribute, but it seems ok for the first version. Agree -> issue?

Reply:
Agree, will create an issue. But I am afraid that the nature of Crudella inherently requires _an identifier_. It seems possible to have it defined as a custom template parameter.

Reply:
#7